thefuck/rules/choco_install.py

```diff
@@ -0,0 +1,29 @@
+def match(command):
+    return (('choco install' in command.script_parts or 'cinst' in command.script_parts)
```

Comment:
```suggestion
    return ((command.script.startswith('choco install') or 'cinst' in command.script_parts)
```

Also, the outer parenthesis can be removed.